### DISCUSSION TITLE
为什么我的邮件里提示我通过了GitHub Models，但是我使用时提示我没有使用权限

### DISCUSSION THREAD

**Discussion by tianzhentech** [community/community#139662] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![image](https://github.com/user-attachments/assets/9b04503a-236c-4972-bc93-458bee46c1df)
![image](https://github.com/user-attachments/assets/e353611a-5b55-4854-98a4-0c2e39f3ac38)

**Comment by Akash1134** [comment]
Hi @tianzhentech , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
